src/DayColumn.js

```diff
@@ -140,12 +140,14 @@ let DaySlot = React.createClass({
       , eventWrapperComponent: EventWrapper
       , rtl: isRtl
       , step
+      , timeslots
```

Comment:
Setting timeslots at TimeGrid.defaultProps will work I think. It's a parent to both TimeColumn and DaySlot, and TimeColumn is a parent for TimeSlotGroup. (But I'm not 100% on React yet so I'm not sure.)